rebar.lock

```diff
@@ -1,4 +1,4 @@
-{"1.1.0",
+{"1.2.0",
```

Comment:
I don't think we want this yet?

Reply:
It's backwards compatible, I figured it was a good a time as any to do it, but I can revert it if you prefer.

Reply:
I don't care too much, @motobob do you?

Reply:
not in this repo. let it be as is 
in our private ones I want to prereserve 1.1.0.

Reply:
eb5351c